CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+Thank you for your interest in contributing to Glosario!
+We welcome contributions of all kinds.
+If you're here to submit a definition,
+either for a new term 
```

Comment:
GitHub preserves newline character, so it would be good to keep paragraphs on one line.